validator-frontend/navigation/MainTabNavigator.js

```diff
@@ -4,12 +4,21 @@ import { AntDesign, Ionicons } from '@expo/vector-icons'
 import HomeScreen from '../screens/HomeScreen'
 import ScanScreen from '../screens/ScanScreen'
 
+import * as Localization from 'expo-localization';
+import i18n from 'i18n-js';
+const en = require('../lang/en.json');
```

Comment:
Can we do this as an import (likely: import * as en from '../lang/en.json')?

Does this work in a production build?